pkg/utils/backoff.go

```diff
@@ -0,0 +1,135 @@
+package utils
+
+import (
+	"fmt"
+	"time"
+)
+
+const (
+	DefaultInitialInterval = 500 * time.Millisecond
+	DefaultMultiplier      = 1.5
+	DefaultMaxInterval     = 60 * time.Second
```

Comment:
Just tested and even with 60 seconds it does not work with JVM env. I would suggest bumping up default max to 300s to cover potential other scenarios.